pkg/scheduler/scheduler.go

```diff
@@ -367,6 +367,20 @@ func (sched *Scheduler) bind(assumed *v1.Pod, b *v1.Binding) error {
 	// If binding succeeded then PodScheduled condition will be updated in apiserver so that
 	// it's atomic with setting host.
 	err := sched.config.GetBinder(assumed).Bind(b)
+	if err != nil {
+		if err := sched.config.SchedulerCache.ForgetPod(assumed); err != nil {
+			glog.Errorf("scheduler cache ForgetPod failed: %v", err)
+		}
+
+		sched.config.Error(assumed, err)
+		sched.config.Recorder.Eventf(assumed, v1.EventTypeWarning, "FailedScheduling", "Binding rejected: %v", err)
```

Comment:
This is not the same event as "Binding Rejected" below. A `GetBinder` error is a (pretty serious) internal error of the scheduler.